packages/webpack-extraction-plugin/src/GriffelCSSExtractionPlugin.test.ts

```diff
@@ -34,6 +34,7 @@ async function compileSourceWithWebpack(
       path: path.resolve(__dirname),
       filename: 'bundle.js',
       pathinfo: false,
+      assetModuleFilename: '[name][ext]',
```

Comment:
Configure explicitly to have original filenames.

Reply:
will end users need to have this in their webpack configs ?

Reply:
Nope, it's just to have the same names for assets in `fs.json` files uses in tests.